scripts/manage/update_management_console.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Looks like I can't suggest a change to the name of this file, but in line with the suggestions further up this should be `update_management_environment.sh`

Reply:
Done.